controllers/customer-admin-controller/Gopkg.toml

```diff
@@ -0,0 +1,54 @@
+# Force dep to vendor the code generators, which aren't imported just used at dev time.
```

Comment:
Why a totally separate place to manage dependencies? I thought we had agreed we were going to manage deps in a single place.

Reply:
This will definitely get placed under the glide file.  I just plopped my code here.  I didn't want to refactor everything if this wasn't the final resting spot.

Reply:
I've taken @jim-minter's advice and removed the controllers `vendor` directory and `gopkg` files.